examples/to_svg.py

```diff
@@ -104,17 +104,19 @@
     size = np.array([10, 20, 20])
     viewer.add_markers(markers, size=size)
 
+    # sample vector coord-like data
     n = 100
-    pos = np.zeros((n, 4), dtype=np.float32)
+    pos = np.zeros((n, 2, 2), dtype=np.float32)
     phi_space = np.linspace(0, 4 * np.pi, n)
     radius_space = np.linspace(0, 100, n)
 
     # assign x-y position
-    pos[:, 0] = radius_space * np.cos(phi_space) + 256
-    pos[:, 1] = radius_space * np.sin(phi_space) + 256
+    pos[:, 0, 0] = radius_space * np.cos(phi_space) + 350
+    pos[:, 0, 1] = radius_space * np.sin(phi_space) + 256
+
     # assign x-y projection
-    pos[:, 2] = 2 * radius_space * np.cos(phi_space)
-    pos[:, 3] = 2 * radius_space * np.sin(phi_space)
+    pos[:, 1, 0] = 2 * radius_space * np.cos(phi_space)
+    pos[:, 1, 1] = 2 * radius_space * np.sin(phi_space)
```

Comment:
Perhaps we should add a `napari.data` module that includes stuff like this? It's kind of annoying to see it repeated (and thus needing updating) everywhere.

Reply:
agreed - but can that come in a separate PR